package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "SSI-SDK-workspace",
   "private": true,
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
Please do not update versions manually. This project is managed through lerna and our CI integration doing automatic versionsing based upon conventional commit messages in Git

Reply:
resolved